src/NuGet.Indexing/RAMDirectoryWrapper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Lucene.Net.Store;
+using System;
+
+namespace NuGet.Indexing
+{
+    // This class wraps the RAMDirectory so we can correct the return of FileModified function
+    public class RAMDirectoryWrapper: RAMDirectory
+    {
+        public RAMDirectoryWrapper() : base() { }
+
+        public RAMDirectoryWrapper(Directory seedDirectory) : base(seedDirectory) { }
+
+        /// <summary>
+        /// Returns the time (as a long) the named file was last modified in a Windows FileTime in UTC
+        /// </summary>
+        /// <param name="name">File Name</param>
+        /// <returns>A long that represents a UTC Windows FileTime</returns>
+        /// <remarks>The implmentation here is to keep in line with the implmentation is AzureDirectory for use in the AzureDirectorySychronizer.
```

Comment:
Also, to reference a class, use `<see cref="AzureDirectory"/>` and `<see cref="AzureDirectorySynchronizer"/>`